folium/features.py

```diff
@@ -227,18 +229,20 @@ def __init__(self, data, style_function=None):
         super(GeoJson, self).__init__()
         self._name = 'GeoJson'
         if 'read' in dir(data):
+            self.embed = True
             self.data = json.load(data)
         elif type(data) is dict:
```

Comment:
Same as above.  I see `isinstance(object, dict)` more that `type(object) is dict`.  Again, not opposed, just curious.